src/main/java/com/jalasoft/webservice/controller/OCRController.java

```diff
@@ -42,14 +45,16 @@ public Response OCRExtractor (@RequestParam("file") MultipartFile file,
                                   @RequestParam(value = "checksum",defaultValue = "false")String checksum,
                                   @RequestParam(value = "lang", defaultValue = "") String lang) throws IOException,
                                   NoSuchAlgorithmException {
+        logger.info("Starting OCR Controller - Method: " + new Object() {}.getClass().getEnclosingMethod().getName());
         String filePath = FileManager.getFilePath(file);
         Checksum checksum1 = new Checksum();
         Response test = new Response();
         DBManager db = new DBManager();
         String checksumResult = checksum1.checksum(filePath);
         String pathDb = "";
-        if (checksum.equals(checksumResult)) {
-            if (db.getPath(checksumResult).isEmpty()) {
+        if (checksum.equals(checksumResult)){
+            logger.error("The cheksum send is not match - Method: " + new Object() {}.getClass().getEnclosingMethod().getName());
```

Comment:
review this line do not overload the 100 character

Reply:
Done